### ISSUE TITLE
Illustratr: Masonry block is properly calculated only after window resize or not shown at all

### ISSUE THREAD

**Issue by daria2303** [Automattic/themes#3778]
## Steps to replicate

1. Install Illustratr
2. Create Masonry block, uploads images
3. Publish
4. Enter the site 

## Result

In the user's report images are loaded with spaces, only after we resize the window JS runs: https://d.pr/v/bULjPP

https://user-images.githubusercontent.com/6737345/144395111-dc80a081-8b58-4437-8cf2-b3140543fa28.mp4

On my test site Masonry Block did't load at all.

## Expected

Masonry is loaded without big spaces between images, without window resize.

* Follow-up ticket: 3969940-zd-woothemes
